frontend/src/store/users/reducers.ts

```diff
@@ -98,12 +127,39 @@ export const reducer = (state: IState = initState, action: Action): IState => {
       ...state,
       user: {
         ...state.user,
-        rsvp: state.user.rsvp.filter(meeting => meeting !== action.payload.meetingAddress),
-        attend: state.user.attend.filter(meeting => meeting !== action.payload.meetingAddress),
-        withdraw: [...state.user.withdraw, action.payload.meetingAddress]
+        data: {
+          ...state.user.data,
+          rsvp: state.user.data.rsvp.filter(meeting => meeting !== action.payload.meetingAddress),
+          attend: state.user.data.attend.filter(meeting => meeting !== action.payload.meetingAddress),
+          withdraw: [...state.user.data.withdraw, action.payload.meetingAddress]
+        }
       }
     };
   }
 
+  if (isType(action, meetingActions.AddMeetingProposal)) {
+    return state;
+
+    // TODO: next version of smart contract (creator of the proposal should automatically vote on it)
+    // return {
+    //   ...state,
+    //   user: {
+    //     ...state.user,
+    //     proposals: [...state.user.proposals, action.payload.proposal]
+    //   }
+    // };
+  }
+
+  if (isType(action, meetingActions.VoteMeetingProposal) ||
+      isType(action, meetingActions.ExecuteMeetingProposal)) {
```

Comment:
I have noticed that when the user executes a proposal then he ends up with a duplicate proposal in the redux store (MongoDB is correctly showing just one updated proposal with Executed state.)

So I propose to remove the trigger for "meetingActions.ExecuteMeetingProposal" for now in the users reducer.

I will review it again myself during refactoring / optimization phase as there seems to be an anti-pattern in our source code which allows for this weird behaviour.